src/wavesurfer.js

```diff
@@ -382,6 +382,22 @@ var WaveSurfer = {
         }
     },
 
+    loadExistingMediaElement: function (elt, peaks) {
```

Comment:
Again, almost a 1-to-1 copy of the `loadMediaElement`. The only difference is which backend method it calls.